diff --git a/library/ftx/ftx_sct_services.pas b/library/ftx/ftx_sct_services.pas
index f376a50..2fecfa9 100644
--- a/library/ftx/ftx_sct_services.pas
+++ b/library/ftx/ftx_sct_services.pas
@@ -5513,6 +5513,7 @@ initialization
   {$IFDEF FPC}
   SNOMED_DATE_FORMAT := DefaultFormatSettings;
   SNOMED_DATE_FORMAT.ShortDateFormat := 'dd/mm/yyyy';
+  SNOMED_DATE_FORMAT.DateSeparator := '/';
   {$ELSE}
   SNOMED_DATE_FORMAT := TFormatSettings.Create('en-AU');
   {$ENDIF}
